raiutils/raiutils/data_processing/data_processing_utils.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import datetime
+import json
+from typing import Any, Dict, List
+
+import numpy as np
+import pandas as pd
+from scipy.sparse import issparse
+from sklearn.utils import check_consistent_length
+
+_DF_COLUMN_BAD_NAME = "DataFrame column names must be strings."\
+    " Name '{0}' is of type {1}"
+_LIST_NONSCALAR = "Lists must be of scalar types"
+_TOO_MANY_DIMS = "Array must have at most two dimensions"
+
+
+def convert_to_list(array, custom_err_msg=None):
```

Comment:
maybe add some docstring here?  I think it is a public method?

Reply:
Done